instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -291,6 +318,14 @@ def process_response(self, request, response):
 
         activation = request.META.pop(self._environ_activation_key, None)
         span = request.META.pop(self._environ_span_key, None)
+        active_requests_count_attrs = request.META.pop(
+            self._environ_active_request_attr_key, None
+        )
+        duration_attrs = request.META.pop(
+            self._environ_duration_attr_key, None
+        )
+        duration_attrs[SpanAttributes.HTTP_STATUS_CODE] = response.status_code
```

Comment:
Null check for the `duration_attrs` is missing.

Reply:
Sure. Added